src/resonanceReconstruction/rmatrix/legacy/resolved/SpinGroup/MultiLevelBreitWigner/test/MultiLevelBreitWigner.test.hpp

```diff
@@ -0,0 +1,120 @@
+SCENARIO( "SpinGroup" ) {
+
+  GIVEN( "valid data for a SpinGroup" ) {
+
+    // test based on Rh105 ENDF/B-VII.1 resolved resonance evaluation
+
+    // scattering radius from equation D.14
+    double a = 0.123 * std::pow( 104.005 * 1.008664, 1. / 3. ) + 0.08;
```

Comment:
This is another one of those places where I, personally, will write "const" if the object, in this case double a, isn't going to change below. It's basically a small nugget of information that tells a reader that they don't have to look around for changes in the value (if indeed it doesn't change!) while they're scanning the code and trying to understand it. I.e., one less thing for somebody to need to think about - and a nice compile-time check that you don't accidentally change something that you didn't intend to change.

Reply:
Yeah, I should do that more rigorously. And the same goes for const ...& variables to avoid copies.